python/lbann/launcher/__init__.py

```diff
@@ -53,6 +55,7 @@ def run(lbann_exe, model, data_reader, optimizer,
         system (str, optional): Target system.
         partition (str, optional): Scheduler partition.
         account (str, optional): Scheduler account.
+        reservation (str, optional): Reservation name (for DATs).
```

Comment:
As far as I can tell, "DAT" is an LLNL-specific term.
```suggestion
        reservation (str, optional): Scheduler advance reservation.
```